core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.html

```diff
@@ -1,14 +1,14 @@
 <dot-form-dialog
-    data-testId="dialogForm"
+    *ngIf="vm$ | async as vm"
     [saveButtonDisabled]="(isFormValid$ | async) === false"
     (cancel)="onCancel()"
     (save)="onSave()"
-    *ngIf="vm$ | async as vm"
+    data-testId="dialogForm"
 >
-    <form class="p-fluid" [formGroup]="form" data-testId="form" *ngIf="form">
+    <form class="p-fluid" *ngIf="form" [formGroup]="form" data-testId="form">
```

Comment:
Why do you need a double `ngIf` here?

Reply:
no idea, that's legacy code, the IDE re-arranged the attributes